recipes/mender-client/debian-master/rules

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 
 %:
-	dh $@
+	dh $@ --with systemd
```

Comment:
Hmm, if I understand the documentation right, this should have already been enabled, since we are using `compat level >= 10`. But obviously not, since it wasn't working before? :thinking:

Reply:
It didn't work without this. The service files were installed but not enabled (iow no symlink from `/etc`)

Reply:
Yeh, I believe you (since obviously it wasn't before). Just wanted to point out that it doesn't match with what I got out of the documentation at least